lib/src/main/java/com/diffplug/spotless/PaddedCell.java

```diff
@@ -28,6 +29,8 @@
 import java.util.Objects;
 import java.util.function.Function;
 
+import org.omg.CORBA.portable.OutputStream;
```

Comment:
Oops, did you mean to import `java.io.OutputStream` rather than `org.omg.CORBA.portable.OutputStream` here?

Reply:
Lol, thanks, good catch!!!  The dangers of not following yagni ;-)